spec/views/tutoring_sessions/index_spec.rb

```diff
@@ -3,15 +3,19 @@
 RSpec.describe 'tutoring_sessions/index', type: :view do
   before(:each) do
     @tutoring_session = create(:tutoring_session, location: 'library',
```

Comment:
I dont think we should be hard coding in values for tests, we should be using the @tutoring_session = create(:tutoring_session)